imports/ui/report-wizard/report-wizard.jsx

```diff
@@ -68,7 +77,13 @@ class ReportWizard extends Component {
         dispatch(replace(match.url.replace(viewMode, enforcedViewMode)))
       }
     }
+    if (!prevProps.reportItem || prevProps.reportItem.title !== reportItem.title) {
+      if (reportItem) {
+        this.setState({reportTitle: reportItem.title})
```

Comment:
@nbiton the temporary display of the old title still persists with the implementation. Is this correct implementation? this.props.reportItem goes through the cycle of dispatch and maybe that delays of setState? I aslo tried to set state in handleSubmit but it did not work as intended... 😢 

> To avoid the effect of the new name switching back to the old name briefly, store the current name in a state variable which will be used for editing and presentation instead of the prop variable, and re-set the state variable to the prop variable's value in componentDidUpdate if (!prevProps.reportItem || prevProps.reportItem.title !== this.props.reportItem.title)